ci-operator/step-registry/openshift-qe/cluster-density/openshift-qe-cluster-density-commands.sh

```diff
@@ -14,13 +14,20 @@ source ./venv_qe/bin/activate
 ES_PASSWORD=$(cat "/secret/password")
 ES_USERNAME=$(cat "/secret/username")
 
+export GSHEET_KEY_LOCATION="${CLUSTER_PROFILE_DIR}/gce.json"
+
+export EMAIL_ID_FOR_RESULTS_SHEET=prubenda@redhat.com
+
 git clone https://github.com/cloud-bulldozer/e2e-benchmarking
 pushd e2e-benchmarking/workloads/kube-burner
 export JOB_ITERATIONS=27
```

Comment:
Same here, if we want to keep with a static number, maybe base it off the number of nodes versus statically assigning 27.

Reply:
Added here but also added as part of some scaling pr I have open: https://github.com/openshift/release/pull/39331